pt-BR/mail/chrome/messenger/pbMessenger.dtd

```diff
@@ -363,14 +363,14 @@
  entities below were automatically added by l10nmerge
 -->
 
-<!ENTITY pbContentTextContextMenu-evernote.label       "Add to Evernote">
-<!ENTITY fullScreenMode.label "Enter Full Screen">
-<!ENTITY pbMessagePaneContext-evernote.label     "Add to Evernote">
+<!ENTITY pbContentTextContextMenu-evernote.label       "Adicionar ao Evernote">
+<!ENTITY fullScreenMode.label "Colocar em tela cheia">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY fullScreenMode.label "Ver em tela cheia">